tests/skinhints/skinhints.pro

```diff
@@ -0,0 +1,6 @@
+CONFIG += testcase qskinny
```

Comment:
This line adds another make target ("make check"), which makes it handy to run all tests in a CI system (Github actions etc.)